src/rsa_primitive_verify_tests.txt

```diff
@@ -0,0 +1,5 @@
+# Big-endian public key parts for signature_rsa_example_public_key.der,
+# encoded in hex.
+
+n = CEA80475324C1DC8347827818DA58BAC069D3419C614A6EA1AC6A3B510DCD72CC516954905E9FEF908D45E13006ADF27D467A7D83C111D1A5DF15EF293771AEFB920032A5BB989F8E4F5E1B05093D3F130F984C07A772A3683F4DC6FB28A96815B32123CCDD13954F19D5B8B24A103E771A34C328755C65ED64E1924FFD04D30B2142CC262F6E0048FEF6DBC652F21479EA1C4B1D66D28F4D46EF7185E390CBFA2E02380582F3188BB94EBBF05D31487A09AFF01FCBB4CD4BFD1F0A833B38C11813C84360BB53C7D4481031C40BAD8713BB6B835CB08098ED15BA31EE4BA728A8C8E10F7294E1B4163B7AEE57277BFD881A6F9D43E02C6925AA3A043FB7FB78D
```

Comment:
The most useful things to test are the misuse-resistance of the API. The reason I initially designed the API to do the parsing itself is to minimize the changes that somebody could screw up; e.g. by swapping `(e, n)` for `(n, e)`. So, the tests I really want to see answer the question of "what happens if `e` and `n` are swapped. It turns out, actually, that the maximum value for `e` is smaller than the minimum value for `n`. We should have test vectors that test this: A test vector with the maximum value of `e` and the minimum value of `n` that passes, a test vector for that value of `e` - 1 and the same `n` that fails, a test tvector for the original value of `e` with original value of `n` 0 - 1 that fails. This will demonstrate what the minimum of `n` and maximum of `e` is. Then, another test vector should swap `(n, e)` as `(e, n)` with the minimum/maximum values of n/e to show that there's no way that swapped parameters would ever cause a problem.

Also, I want to see test vectors for leading zeros and for values of `n` and `e` that have the highest bit set. This will demonstrate that the API doesn't enforce the ASN.1 semantics for leading zeros. So, for example, we should show that `n = 00<whatever>` is equivalent to `n = <whatever>` and that `n = FF<whatever>` is equivalent to `00FF<whatever>`. Or, if we want to require that there's no leading zeros, then that needs to be documented and also tested.